src/cf.data.pre

```diff
@@ -10457,4 +10457,99 @@ DOC_START
 		server_pconn_for_nonretriable allow SpeedIsWorthTheRisk
 DOC_END
 
+NAME: happy_eyeballs_connect_timeout
+COMMENT: (msec)
+TYPE: int
+DEFAULT: 250
+LOC: Config.happyEyeballs.connect_timeout
+DOC_START
+	This Happy Eyeballs (RFC 8305) tuning directive specifies the minimum
+	delay between opening a primary to-server connection and opening a
+	spare to-server connection for the same master transaction. This delay
+	is similar to the Connection Attempt Delay in RFC 8305, but it is only
+	applied to the first spare connection attempt. Subsequent spare
+	connection attempts use happy_eyeballs_connect_gap, and primary
+	connection attempts are not artificially delayed at all.
+
+	Terminology: The "primary" and "spare" designations are determined by
+	the order of DNS answers received by Squid: If Squid DNS AAAA query
```

Comment:
Not according to this description. Both cases explained below use use v6 as primary.

Reply:
> Not according to this description

Fixed 4-vs-6 typo in 56dcb65.

> RFC 6540: IPv6 support must be equivalent or better in quality and functionality when compared to IPv4 support in a new or updated IP implementation.

The new Squid code does not discriminate any IP address family so it is compliant with the RFC text you quoted.